cnr_scene_manager_test:
    object_geometries:
        sphere: {sphere: 0.1, color: [1,0.0,0.0,0.5], position: [0.0,0.0,0.0], quaternion: [0, 0, 0, 1]}
        box       : {box: [0.2, 0.2, 0.45], color: [0.75,0.75,0.75,1], position: [0.0,0.0,0.0], quaternion: [0, 0, 0, 1]}
        plane      : {box: [5.0, 5.0, 0.10], color: [0.75,0.75,0.75,1], position: [0.0,0.0,0.0], quaternion: [0, 0, 0, 1]}
#      sidewall   : {mesh: "package://cnr_scene_manager/example_meshes/SidewallPanel.stl", color: [255,0,0,1]}
    scene_objects:
        - {type: "box", frame: "world", position: [0.5,0.5,0.5], quaternion: [0, 0, 0, 1]}
